core/src/test/java/com/google/cloud/sql/core/CoreSocketFactoryTest.java

```diff
@@ -364,6 +365,67 @@ public void testGetApplicationNameFailsAfterInitialization() {
         IllegalStateException.class, () -> CoreSocketFactory.setApplicationName("sample-app"));
   }
 
+  @Test
+  public void create_successfulNamedConnection() throws IOException, InterruptedException {
+
+    FakeSslServer sslServer = new FakeSslServer();
+    int port = sslServer.start(PUBLIC_IP);
+    ApiFetcherFactory factory =
+        new StubApiFetcherFactory(fakeSuccessHttpTransport(Duration.ofSeconds(0)));
+
+    CoreSocketFactory registry =
```

Comment:
Let's be consistent with names and types. Unfortunately, we did not name this ConnectorRegistry, and so should stick with what we have.

Reply:
Fixed. the test variables are now named `coreSocketFactory`.